index.ts

```diff
@@ -35,82 +36,42 @@ const serve = async ({ target, routeFile, host, port, index, build, cache }) =>
 }
 
 const build = async ({ routeFile, outputDir }) => {
-  // Setup bundler
-  const bundler = new Parcel({
-    entries: routeFile,
-    defaultConfig: '@parcel/config-default',
-    mode: 'production',
-    shouldDisableCache: true,
-    targets: {
-      'main': {
-        context: 'node',
-        distDir: outputDir,
-        engines: {
-          node: ">=10"
-        },
-        scopeHoist: false,
-        optimize: false,
-      }
-    }
-  })
-
+  const startTime = new Date().getTime()
```

Comment:
We could eventually come up with a neater pattern for measuring the time. Maybe a decorator function?